openquake/hazardlib/correlation.py

```diff
@@ -20,15 +20,20 @@
 import abc
 import numpy
 
+from nose.tools import set_trace
 
-class BaseCorrelationModel(metaclass=abc.ABCMeta):
+from openquake.hazardlib.imt import SA, PGA
+from openquake.baselib.python3compat import with_metaclass
+
+
+class BaseCorrelationModel(with_metaclass(abc.ABCMeta)):
```

Comment:
`with_metaclass` is not needed anymore, we have move to Python 3, so just use the syntax `metaclass=abc.ABCMeta`

Reply:
Ok, I'll change this